internal/services/v1/experimental.go

```diff
@@ -154,6 +154,12 @@ func (a *bulkLoadAdapter) Next(_ context.Context) (*tuple.Relationship, error) {
 		a.currentBatch = batch.Relationships
 		a.numSent = 0
 
+		for _, rel := range batch.Relationships {
+			if rel.OptionalExpiresAt != nil {
+				return nil, fmt.Errorf("expiration time is not currently supported")
+			}
+		}
```

Comment:
Is this extra defensiveness for the current release?

Reply:
Yep!